src/DebtToken.sol

```diff
@@ -212,16 +202,10 @@ abstract contract DebtToken is ERC4626 {
     /**
      * @notice Modification of the standard ERC4626 transferFrom implementation.
      * @dev No public transferFrom allowed.
+     * @dev The functions approve() and permit() will not revert, but since transferFrom() reverts,
```

Comment:
If they're gone it will revert right?

Reply:
No, still implemented in the contracts it inherits from